### PR TITLE
Accuracy Test Script

### PR DESCRIPTION
**Pull request by Ta7ar**
Script to generate synthetic data based on `sklearn` `iris` and `Wisconsin breast cancer` datasets and subsequently measure synthetic data accuracy by using `RandomForestClassifier` (trained on original data) predictions as ground truth.

### CODE REVIEW

**Review by taylorfturner — APPROVED** [review]

**Comment by taylorfturner** [comment]
add AUC actually to this as well

**Comment by Ta7ar** [comment]
Changes made in the latest commit:
* Use AUC ROC score as a metric instead of calculating percentage of correct predictions.
* Calculate accuracy metrics on both Iris and Wisconsin breast cancer dataset. Previously had to manually change what dataset to calculate accuracy metrics on.
* `with io.capture_output() as captured:` used to suppress unneeded `DataProfiler` print function outputs. 

Comments on performance:
* The AUC ROC score on the Iris dataset is typically between 0.85 to 0.92
* The AUC ROC score on the breast cancer dataset is typically between 0.67 to 0.8

(_The stated performances were based purely on preliminary inspection. Actual performance ranges could differ_)

**Review by taylorfturner — APPROVED** [review]

**Review by tazitoo — APPROVED** [review]
